components/button/button-icon.js

```diff
@@ -104,6 +104,9 @@ class ButtonIcon extends PropertyRequiredMixin(ThemeMixin(ButtonMixin(VisibleOnA
 				:host([h-align="text"]) button {
 					inset-inline-start: var(--d2l-button-icon-h-align);
 				}
+				:host([h-align="text-end"]) button {
```

Comment:
Almost wish we could go back and define `text-start` to align with logical property naming, but I guess that ship has sailed.

Reply:
I know, I thought the same.